etl/steps/data/grapher/artificial_intelligence/2024-06-28/ai_private_investment.py

```diff
@@ -0,0 +1,66 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("ai_investment")
+
+    # Read table from garden dataset.
+    tb = ds_garden["ai_investment"].reset_index()
+
+    industries = {
+        "ai_infrastructure_research_governance": "AI infrastructure, research, and governance",
```

Comment:
Could this one be somewhat shorter, like AI infrastructure and governance? I think it would look better in the faceted chart